spark-3.1-spanner-lib/src/test/java/com/google/cloud/spark/spanner/SparkFilterUtilsTest.java

```diff
@@ -128,26 +98,18 @@ public void testNewFilterBehaviourWithFilterOption() {
 
   @Test
   public void testNewFilterBehaviourNoFilterOption() {
-    if (dataFormat == ARROW && !pushAllFilters) {
-      // If pushAllFilters isn't true, the following test won't pass for ARROW.
-      return;
-    }
-    checkFilters(
-        pushAllFilters, dataFormat, "", "(`a` > 2)", Optional.empty(), GreaterThan.apply("a", 2));
+    checkFilters(pushAllFilters, "", "(`a` > 2)", Optional.empty(), GreaterThan.apply("a", 2));
```

Comment:
Does this work ? pushAllFilters is false and should return in the original version